ProjectLighthouse.Servers.Website/Pages/NotificationsPage.cshtml

```diff
@@ -0,0 +1,139 @@
+@using LBPUnion.ProjectLighthouse.Types.Notifications
+@model LBPUnion.ProjectLighthouse.Servers.Website.Pages.NotificationsPage
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+@{
+    Layout = "Layouts/BaseLayout";
+    Model.Title = Model.Translate(GeneralStrings.Notifications);
+}
+
+@if (Model.User != null && Model.User.IsAdmin)
+{
+    <div class="ui red segment">
+        <h3>Post New Announcement</h3>
+        @if (!string.IsNullOrWhiteSpace(Model.Error))
+        {
+            @await Html.PartialAsync("Partials/ErrorModalPartial", (Model.Translate(GeneralStrings.Error), Model.Error), ViewData)
+        }
+        <form id="form" method="POST" class="ui form center aligned" action="/notifications">
+            @Html.AntiForgeryToken()
+            <div class="field">
+                <label style="text-align: left" for="title">Announcement Title</label>
+                <input type="text" name="title" id="title">
+            </div>
+            <div class="field">
+                <label style="text-align: left" for="content">Announcement Content</label>
+                <textarea name="content" id="content" spellcheck="false" rows="3"></textarea>
+            </div>
+            <button class="ui button green" type="submit" tabindex="0">Post Announcement</button>
+        </form>
+    </div>
+}
+
+@if (Model.Announcements.Any())
+{
+    @foreach (WebsiteAnnouncementEntity announcement in Model.Announcements)
+    {
+        <div class="ui blue segment" style="position: relative;">
+            <div>
+                <h3>@announcement.Title</h3>
+                <div style="padding-bottom: 2em;">
+                    <span style="white-space: pre-line; ">@announcement.Content</span>
+                </div>
+                @if (announcement.Publisher != null)
+                {
+                    <div class="ui tiny bottom left attached label">
+                        Posted by
+                        <a style="color: black" href="~/user/@announcement.Publisher.UserId">
+                            @announcement.Publisher.Username
+                        </a>
+                    </div>
+                }
+            </div>
+            @if (Model.User != null && Model.User.IsAdmin)
+            {
+                <form method="post">
+                    @Html.AntiForgeryToken()
+                    <button
+                        asp-page-handler="delete"
+                        asp-route-type="announcement"
+                        asp-route-id="@announcement.AnnouncementId"
+                        onclick="return confirm('Are you sure you want to delete this announcement?')"
+                        class="ui red icon button"
+                        style="position: absolute; right: 0.5em; top: 0.5em">
+                        <i class="trash icon"></i>
+                    </button>
+                </form>
+            }
+        </div>
+    }
+}
+
+@if (Model.Notifications != null && Model.Notifications.Any())
```

Comment:
```suggestion
@if (Model.Notifications != null && Model.Notifications.Count > 0)
```
Since Notifications is a List, I don't think you need to use LINQ here